axe-testing/whitelist-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "blacklistUrls": [
+    "http://127.0.0.1:8080/termsAndConditions",
+    "http://127.0.0.1:8080/signin?post_auth_redirect=%2F&message=SessionRequired"
+  ],
```

Comment:
I'd recommend blacklisting path patterns, rather than URLs. The base might be dependent on env (port might change), and there will be paths with parameterized portions which we might want to blacklist (e.g. the `post_auth_redirect` query parameter won't always be empty, or we might want to blacklist a route like `manage-collection/${collection id}/edit-column/${column id}`; could do both of these with regex patterns).